packages/test-utils/src/throwingInstance.ts

```diff
@@ -6,7 +6,9 @@ import * as chai from "chai";
  */
 export function throwingInstance<TMock extends object>(target: TMock) {
     const stubbedMethods: Array<string> = Object.keys(
-        (target as any)["tsMockitoInstance"]["mocker"]["methodStubCollections"]
+        (target as any)["tsMockitoInstance"]["mocker"]
+            ? (target as any)["tsMockitoInstance"]["mocker"]["methodStubCollections"]
+            : (target as any)["mocker"]()["mocker"]["methodStubCollections"]
```

Comment:
I have no idea what's happening here - is it easy to explain in a comment?

Reply:
Added a comment